src/app/pages/apps/components/installed-apps/installed-apps.component.ts

```diff
@@ -1,9 +1,53 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import {
+  Component, ChangeDetectionStrategy, OnInit, ChangeDetectorRef,
+} from '@angular/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { ChartRelease } from 'app/interfaces/chart-release.interface';
+import { ApplicationsService } from 'app/pages/apps-old/applications.service';
```

Comment:
I am unsure if we will use the app service from the old apps module. There is a new one - [app/pages/apps/services/applications.service.ts](https://github.com/truenas/webui/blob/master/src/app/pages/apps/services/applications.service.ts)